src/managers/GuildManager.js

```diff
@@ -163,14 +164,21 @@ class GuildManager extends BaseManager {
     } = {},
   ) {
     icon = await DataResolver.resolveImage(icon);
-    if (typeof verificationLevel !== 'undefined' && typeof verificationLevel !== 'number') {
-      verificationLevel = VerificationLevels.indexOf(verificationLevel);
+    if (verificationLevel) {
+      verificationLevel =
+        typeof verificationLevel === 'number' ? verificationLevel : VerificationLevels[verificationLevel];
```

Comment:
You could probably change this to:
```suggestion
    if (typeof verificationLevel === 'string') {
      verificationLevel = VerificationLevels[verificationLevel];
```

And same for the others, those values take either a string or a number, one is untouched, and the other is transformed.

Reply:
Same goes fo the other checks.